src/index.js

```diff
@@ -118,6 +79,9 @@ ipcRenderer.on('export', (event, message) => {
 
 ipcRenderer.on('export-tags', (event, exportConfig) => {
   addLoader();
+  if(videotagging.imagelist){
+    videotagging.imagelist = videotagging.imagelist.map((filepath) => {return path.join(videotagging.sourceDir,filepath)})
```

Comment:
This could be simplified to the following

```
videotagging.imagelist = videotagging.imagelist.map(filepath => path.join(videotagging.sourceDir,filepath))
```